web/source/kmwembedded.ts

```diff
@@ -13,7 +13,7 @@
 /*****************************************/
 
 namespace com.keyman.osk {
-  VisualKeyboard.prototype.optionKey = function(this: VisualKeyboard, e: KeyElement, keyName: string, keyDown: boolean) {
+   VisualKeyboard.prototype.optionKey = function(this: VisualKeyboard, e: KeyElement, keyName: string, keyDown: boolean) {
```

Comment:
I know this is whitespace only, but can we avoid any changes to kmwembedded.ts -- please revert it.